packages/devtools_app/lib/src/screens/network/network_request_inspector.dart

```diff
@@ -50,15 +51,25 @@ class NetworkRequestInspector extends StatelessWidget {
       valueListenable: controller.selectedRequest,
       builder: (context, data, _) {
         final tabs = <DevToolsTab>[
-          _buildTab(NetworkRequestInspector._overviewTabTitle),
+          _buildTab(tabName: NetworkRequestInspector._overviewTabTitle),
           if (data is DartIOHttpRequestData) ...[
-            _buildTab(NetworkRequestInspector._headersTabTitle),
+            _buildTab(tabName: NetworkRequestInspector._headersTabTitle),
             if (data.requestBody != null)
-              _buildTab(NetworkRequestInspector._requestTabTitle),
+              _buildTab(tabName: NetworkRequestInspector._requestTabTitle),
             if (data.responseBody != null)
-              _buildTab(NetworkRequestInspector._responseTabTitle),
+              _buildTab(
+                tabName: NetworkRequestInspector._responseTabTitle,
+                trailing: Row(
```

Comment:
You shouldn't need to put CopyToClipboardControl in a Row. It should already be aligned right by the DevToolsTab widget.

Reply:
![image](https://user-images.githubusercontent.com/1386322/173423244-f16ec8b5-1124-4360-8036-9e46ecc33701.png)
No dice unfortunately :(